tests/plugins/test_plugin.py

```diff
@@ -17,6 +17,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# [START plugins_interface]
```

Comment:
It should be "plugin_example"

Reply:
I use conventions "[file name] _ [section name]". This allows us to easily find the place where the fragment is loaded.